src/PasswordField.js

```diff
@@ -20,15 +20,12 @@ const getStyles = (props, context, state) => {
 
   const styles = {
     root: {
-      display: 'flex',
-      width: props.fullWidth ? '100%' : null
-    },
-    inputContainer: {
-      flex: 1
+      position: 'relative',
+      display: props.fullWidth ? 'block' : 'inline-block'
     },
     input: {
-      width: '100%',
-      paddingRight: 56
+      paddingRight: 56,
+      boxSizing: 'border-box'
```

Comment:
weird, it seems to work for me! [msdn says IE8+](https://msdn.microsoft.com/en-us/library/dd183522(v=vs.85).aspx)